config/clouddriver.yml

```diff
@@ -40,3 +40,20 @@ google:
     - name: ${providers.google.primaryCredentials.name}
       project: ${providers.google.primaryCredentials.project}
       jsonPath: ${providers.google.primaryCredentials.jsonPath}
+
+cf:
+  enabled: ${providers.cf.enabled:false}
+
+  # TODO: Migrate this out of clouddriver and into either orca or igor
+  jenkins:
```

Comment:
Did you address @ewiseblatt comments on the other PR about having these settings here? Are they still needed ( jenkins )